### ISSUE TITLE
Customize stair generation chance

### ISSUE THREAD

**Issue by igroglaz** [draconisPW/PWMAngband#34]
Tangaria got
```
# Option: restrict the use of stairs.
# 1 = up/down staircases are not connected between levels
LIMIT_STAIRS = 1
```
This is beautiful option, but with huge level generated sometimes it takes very long time to find entrance to the next level, which could be boring on 'dull' levels. In most cases there is only one stair down generated on lvl.

It would be fun to add customization for numbers of stair in constants.txt

**Comment by draconisPW** [comment]
Hard to do because each level type has different number of stairs. For example, labyrinth only has one while normal has 3-5. Maybe add this in dungeon_profile.txt, should be possible.

**Comment by draconisPW** [comment]
Adding this for 1.3 since I will have to release another bugfix version.

**Comment by draconisPW** [comment]
Implemented in eca1c1b.
